src/components/LandingPage.vue

```diff
@@ -35,91 +35,86 @@
+              {{ $t("app.main.addons" /* Addons */) }}
+            </label>
+            <span style="margin-left: 5px" />
+            <Button
+              v-for="(addon, index) in allAddonConfigs"
+              :key="index"
+              type="addon"
+              :class="{ active: addonSelected === addon.addonName }"
+              :disabled="!addon.isInstalled"
+              @click="addonSelected = addon.addonName"
             >
-            </Dropdown>
+              {{ addon.addonName }}
+            </Button>
           </div>
-        </div>
-        <div
-          v-if="configStep === 0 && allAddonConfigs.length > 1"
-          id="addonbttns"
-        >
-          <label :key="addonSelected" class="btn-label">
-            {{ $t("app.main.addons" /* Addons */) }}
-          </label>
-          <span style="margin-left: 5px" />
-          <Button
-            v-for="(addon, index) in allAddonConfigs"
-            :key="index"
-            type="addon"
-            :class="{ active: addonSelected === addon.addonName }"
-            :disabled="!addon.isInstalled"
-            @click="addonSelected = addon.addonName"
-          >
-            {{ addon.addonName }}
-          </Button>
-        </div>
-        <div v-if="configStep === 0" id="dashboard">
-          <RefreshButton
-            :is-settings-ok="config.wowpath.valided"
-            :is-version-selected="versionSelected"
-            :is-account-selected="accountSelected"
-            :is-sv-ok="WeakAurasSaved() || PlaterSaved()"
-            :fetching="fetching"
-            :last-update="accountSelected && accountSelected.lastWagoUpdate"
-            :auras-shown="
-              config.showAllAuras
-                ? aurasSortedForView.length
-                : aurasWithUpdateSortedForView.length
-            "
-          ></RefreshButton>
-          <br />
-          <div
-            id="aura-list"
-            :class="{
-              hidden: config.showAllAuras
-                ? aurasSortedForView.length <= 0
-                : aurasWithUpdateSortedForView.length <= 0,
-            }"
-          >
-            <Aura
-              v-for="aura in config.showAllAuras
-                ? aurasSortedForView
-                : aurasWithUpdateSortedForView"
-              :key="aura.slug"
-              :aura="aura"
-              :show-all-auras="config.showAllAuras"
-            ></Aura>
+          <div id="dashboard">
+            <RefreshButton
+              :is-settings-ok="config.wowpath.valided"
+              :is-version-selected="versionSelected"
+              :is-account-selected="accountSelected"
+              :is-sv-ok="WeakAurasSaved() || PlaterSaved()"
+              :fetching="fetching"
+              :last-update="accountSelected && accountSelected.lastWagoUpdate"
+              :auras-shown="aurasSortedForView.length"
+            ></RefreshButton>
+            <br />
+            <AuraHeaders
```

Comment:
Headers should be hidden when there is no aura to show
`:class="{ hidden: aurasSortedForView.length <= 0 }"`

![image](https://user-images.githubusercontent.com/189656/95924209-718a5700-0db7-11eb-8cbb-f659f7e18047.png)

Reply:
I was thinking about creating a component `AuraList` holding both `AuraHeaders` and `AuraListItem`, which would take care of the sorting and would not "pollute" the code of the `LandingPage`. The only thing is that for now, the sort parameters would be held inside of the list component, so if we change page and go on Settings or others, the sort parameters would reset. Would that be acceptable ? 
That would be until we could refactor the config object into something more global, and not held by the `LandingPage`.